memlab.bib

```diff
@@ -7552,7 +7644,7 @@ @article{LyonEtal08
 	Title = {A computational model of spatial visualization capacity},
 	Volume = {57},
 	Year = {2008},
-	Bdsk-File-1 = {YnBsaXN0MDDSAQIDBFxyZWxhdGl2ZVBhdGhZYWxpYXNEYXRhXxAeMjAwOC1seW9uX2d1bnplbG1hbm5fZ2x1Y2sucGRmTxEB4gAAAAAB4gACAAANbm91dmVhdU1hY1BybwAAAAAAAAAAAAAAAAAAyedG5EgrAAADEilwHjIwMDgtbHlvbl9ndW56ZWxtYW5uX2dsdWNrLnBkZgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAIjVzJTH5YAAAAAAAAAAAAAQACAAAJIAAAAAAAAAAAAAAAAAAAAAhtYWdlbGxhbgAQAAgAAMnnfyQAAAARAAgAAMlMxKgAAAABABADEilwAAjFJwAIe3AAAJigAAIAVW5vdXZlYXVNYWNQcm86VXNlcnM6AHJvYmVydHNla3VsZXI6AERyb3Bib3g6AG1hZ2VsbGFuOgAyMDA4LWx5b25fZ3VuemVsbWFubl9nbHVjay5wZGYAAA4APgAeADIAMAAwADgALQBsAHkAbwBuAF8AZwB1AG4AegBlAGwAbQBhAG4AbgBfAGcAbAB1AGMAawAuAHAAZABmAA8AHAANAG4AbwB1AHYAZQBhAHUATQBhAGMAUAByAG8AEgBDVXNlcnMvcm9iZXJ0c2VrdWxlci9Ecm9wYm94L21hZ2VsbGFuLzIwMDgtbHlvbl9ndW56ZWxtYW5uX2dsdWNrLnBkZgAAEwABLwAAFQACABT//wAAAAgADQAaACQARQAAAAAAAAIBAAAAAAAAAAUAAAAAAAAAAAAAAAAAAAIr},
+	Bdsk-File-1 = {YnBsaXN0MDDUAQIDBAUGJCVYJHZlcnNpb25YJG9iamVjdHNZJGFyY2hpdmVyVCR0b3ASAAGGoKgHCBMUFRYaIVUkbnVsbNMJCgsMDxJXTlMua2V5c1pOUy5vYmplY3RzViRjbGFzc6INDoACgAOiEBGABIAFgAdccmVsYXRpdmVQYXRoWWFsaWFzRGF0YV8QHjIwMDgtbHlvbl9ndW56ZWxtYW5uX2dsdWNrLnBkZtIXCxgZV05TLmRhdGFPEQHiAAAAAAHiAAIAAA1ub3V2ZWF1TWFjUHJvAAAAAAAAAAAAAAAAAADJ50bkSCsAAAMSKXAeMjAwOC1seW9uX2d1bnplbG1hbm5fZ2x1Y2sucGRmAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAiNXMlMflgAAAAAAAAAAAABAAIAAAkgAAAAAAAAAAAAAAAAAAAACG1hZ2VsbGFuABAACAAAyed/JAAAABEACAAAyUzEqAAAAAEAEAMSKXAACMUnAAh7cAAAmKAAAgBVbm91dmVhdU1hY1BybzpVc2VyczoAcm9iZXJ0c2VrdWxlcjoARHJvcGJveDoAbWFnZWxsYW46ADIwMDgtbHlvbl9ndW56ZWxtYW5uX2dsdWNrLnBkZgAADgA+AB4AMgAwADAAOAAtAGwAeQBvAG4AXwBnAHUAbgB6AGUAbABtAGEAbgBuAF8AZwBsAHUAYwBrAC4AcABkAGYADwAcAA0AbgBvAHUAdgBlAGEAdQBNAGEAYwBQAHIAbwASAENVc2Vycy9yb2JlcnRzZWt1bGVyL0Ryb3Bib3gvbWFnZWxsYW4vMjAwOC1seW9uX2d1bnplbG1hbm5fZ2x1Y2sucGRmAAATAAEvAAAVAAIAFP//AACABtIbHB0eWiRjbGFzc25hbWVYJGNsYXNzZXNdTlNNdXRhYmxlRGF0YaMdHyBWTlNEYXRhWE5TT2JqZWN00hscIiNcTlNEaWN0aW9uYXJ5oiIgXxAPTlNLZXllZEFyY2hpdmVy0SYnVHJvb3SAAQAIABEAGgAjAC0AMgA3AEAARgBNAFUAYABnAGoAbABuAHEAcwB1AHcAhACOAK8AtAC8AqICpAKpArQCvQLLAs8C1gLfAuQC8QL0AwYDCQMOAAAAAAAAAgEAAAAAAAAAKAAAAAAAAAAAAAAAAAAAAxA=},
```

Comment:
Should be removed...not sure what this is

Reply:
This is a file in the bibtex entry that links to the article online. I've removed it in the latest commit